cloud/ec2.go

```diff
@@ -789,6 +789,18 @@ func (m *ec2Manager) TerminateInstance(ctx context.Context, h *host.Host, user,
 	if !IsEC2InstanceID(h.Id) {
 		return errors.Wrap(h.Terminate(ctx, user, fmt.Sprintf("detected invalid instance ID '%s'", h.Id)), "terminating instance in DB")
 	}
+
+	if h.NoExpiration {
+		// Clean up remaining DNS records for unexpirable hosts.
+		grip.Error(message.WrapError(deleteHostPersistentDNSName(ctx, m.env, h, m.client), message.Fields{
+			"message":    "could not delete host's persistent DNS name",
+			"op":         "delete",
+			"dashboard":  "evergreen sleep schedule health",
```

Comment:
I haven't seen the "dashboard" field in logs before. How do you plan to use it?

Reply:
I'm mostly using it prevent somebody from randomly deciding to delete or modify the log without knowing what effects it'll have. You can see the dashboard in Splunk (it's called "Evergreen Sleep Schedule Health")

Reply:
thank you!